InternProject/app/Repositories/UserRepository.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App\Repositories;
+
+use App\Models\User;
+use App\Repositories\BaseRepository;
+use App\Helpers\Constant;
+
+class UserRepository extends BaseRepository
+{
+
+    public function getModel()
+    {
+        return User::class;
+    }
+
+    public function getMale()
+    {
+        //dd(User::MALE);
+        //$males = User::where('gender',User::MALE)->get(); //dùng const của model
+        $males = User::where('gender',Constant::GENDER['MALE'])->get(); //dùng const của constant
```

Comment:
$males = User::where('gender',Constant::GENDER['MALE'])->get(); //dùng const của constant

```suggestion
    $males = $this->model->where('gender',Constant::GENDER['MALE'])->get(); //dùng const của constant
```